CMakeLists.txt

```diff
@@ -486,6 +505,10 @@ IF(LCMS_SUPPORT_CAN_BE_COMPILED)
     TARGET_LINK_LIBRARIES(raw ${LCMS2_LIBRARIES})
 ENDIF()
 
+IF(ZLIB_FOUND)
+    TARGET_LINK_LIBRARIES(raw ${ZLIB_LIBRARY})
```

Comment:
Please change to ZLIB_LIBRARIES. See https://cmake.org/cmake/help/v3.14/module/FindZLIB.html.